pasteit/routes/index.js

```diff
@@ -1,24 +1,45 @@
 var express = require('express');
 var bodyParser = require("body-parser");
 var router = express();
+var redis = require("redis");
+var client = redis.createClient();
+var flatten = require("flat")
 
-router.use(bodyParser.urlencoded({ extended : false }));
-router.use(bodyParser.json());
-router.use(function(err, req, res, next) {
-    console.error(err.stack);
-    res.status(500).send("Something broke!");
-});
+var getNewHash = function() {
```

Comment:
Why not use a UUID4?

Reply:
We can use that as well, no need to use my function. I'll patch it up.

Reply:
UUID4 is now used

![grafik](https://user-images.githubusercontent.com/8678523/31322552-5c675ff2-ac9a-11e7-8de4-62a336b6115c.png)